spyder/plugins/history.py

```diff
@@ -111,7 +116,8 @@ def __init__(self, parent):
         # Find/replace widget
         self.find_widget = FindReplace(self)
         self.find_widget.hide()
-        self.register_widget_shortcuts(self.find_widget)
+        if not self.testing:
+            self.register_widget_shortcuts(self.find_widget)

```

Comment:
I think It's better to monkeypath this in the test fixture instead of adding test logic to the plugin:

```
@pytest.fixture
def historylog(qtbot, monkeypatch):
    monkeypatch.setattr(
            'spyder.plugins.history.HistoryLog.register_widget_shortcuts',
            lambda *args: None)

    historylog = history.HistoryLog(None, testing=True)
    [...]

```

Please also remove the `testing` arg, and monkeypath the fixture

Reply:
Thanks!  I had added `testing` based on `test_ipythonconsole`, but I agree that the monkeypatch is better.